### PR TITLE
Package gremlin.0.1.1

### PR DESCRIPTION
**Pull request by bramford**
### `gremlin.0.1.1`
Gremlin Client Library
This is an Apache Tinkerpop3 Gremlin client library.

See the official tinkerpop3 and gremlin docs:
- http://tinkerpop.apache.org/docs/current/

This client library is implemented following the driver provider requirements:
- http://tinkerpop.apache.org/docs/current/dev/provider/#_graph_driver_provider_requirements

---
* Homepage: https://github.com/bramford/ocaml-gremlin
* Source repo: git+https://github.com/bramford/ocaml-gremlin
* Bug tracker: https://github.com/bramford/ocaml-gremlin/issues

---
:camel: Pull-request generated by opam-publish v2.0.0

### CODE REVIEW

**Comment by kit-ty-kate** [comment]
You forgot to provide the url to the archive of your package (see the `url` block in other opam files)
```
#=== ERROR while compiling gremlin.v0.1.1 =====================================#
# context              2.0.4 | linux/x86_64 | ocaml-base-compiler.4.08.0 | pinned
# path                 ~/.opam/4.08/.opam-switch/build/gremlin.v0.1.1
# command              ~/.opam/4.08/bin/dune build -j 2 -p gremlin
# exit-code            1
# env-file             ~/.opam/log/gremlin-15-b64e6b.env
# output-file          ~/.opam/log/gremlin-15-b64e6b.out
### output ###
# Error: I don't know about package gremlin (passed through --only-packages/--release)
```

**Comment by kit-ty-kate** [comment]
Also, could you avoid the `v` prefix to the version number? With the notable exception of janstreet packages, there is no need for it and it's rather unwanted and source of bug.

File: packages/gremlin/gremlin.v0.1.1/opam [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+dev-repo: "git+https://github.com/bramford/ocaml-gremlin"
+doc: "https://bramford.github.io/ocaml-gremlin/doc"
+license: "ISC"
+build: [ "dune" "build" "-j" jobs "-p" name ]
+depends: [
+  "ocaml" {>= "4.07.1"}
+  "conduit-lwt-unix"
+  "containers"
+  "core"
+  "dune"
```
Comment (kit-ty-kate) [inline packages/gremlin/gremlin.v0.1.1/opam]:
```suggestion
  "dune" {>= "1.10"}
```
Your `dune-project` file in the source code says you require at least dune 1.10.

**Comment by kit-ty-kate** [comment]
You have a lot of big dependencies with no constraints on their minimal version required. I suspect there are several needed here but if you don't have time to put them all it's ok.

**Comment by kit-ty-kate** [comment]
wait you had a whole lot of constraints in https://github.com/ocaml/opam-repository/pull/14498, where did there all go?

**Comment by kit-ty-kate** [comment]
Also I'm not entirely sure why you removed this PR to create another one. It's more difficult to work with several PRs than with just one if you push over it.

**Comment by bramford** [comment]
Everything should be working now.

> wait you had a whole lot of constraints in #14498, where did there all go?

I've put them back in. I wasn't sure whether to specify the known working dependencies or leave them blank and let others test/report. Different packages from the community do it differently

> Also I'm not entirely sure why you removed this PR to create another one. It's more difficult to work with several PRs than with just one if you push over it.

This was because github doesn't allow branch renaming in PRs as far as I could tell. I wanted the branch name to match the version number.

**Comment by kit-ty-kate** [comment]
Your package is again missing the `url` field

**Comment by bramford** [comment]
> Your package is again missing the `url` field

Fixed.

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**